index.test.js

```diff
@@ -93,6 +94,9 @@ describe('lonlat', () => {
     }, {
       calculated: ll.fromString(str),
       description: 'String with comma separating lon and lat'
```

Comment:
Might be worth adding `, respectively` to this description and the one below for even more clarity.